lib/data/monorepo.json

```diff
@@ -47,7 +47,10 @@
     "aws-sdk-client-mock": "https://github.com/m-radzikowski/aws-sdk-client-mock",
     "aws-sdk-go": "https://github.com/aws/aws-sdk-go",
     "aws-sdk-go-v2": "https://github.com/aws/aws-sdk-go-v2",
-    "aws-sdk-js-v3": "https://github.com/aws/aws-sdk-js-v3",
+    "aws-sdk-js-v3": [
+      "https://github.com/aws/aws-sdk-js-v3",
+      "https://github.com/smithy-lang/smithy-typescript"
```

Comment:
This is not right

Reply:
What's the suggested approach?

I followed the same strategy as with `aws-sdk-rust` monorepo:

```
    "aws-sdk-rust": [
      "https://github.com/smithy-lang/smithy-rs",
      "https://github.com/awslabs/smithy-rs",
      "https://github.com/awslabs/aws-sdk-rust"
    ],
```

Reply:
we probably should check all our monorepos and validate if they share same version or absolutely need to be grouped. we should remove all which don't share same version and don't need to be merged together.